pandas/io/tests/test_clipboard.py

```diff
@@ -71,6 +71,7 @@ def check_round_trip_frame(self, data_type, excel=None, sep=None):
     def test_round_trip_frame_sep(self):
```

Comment:
Do you think you can add round-trip tests for slightly unusual delimiters instead of the standard white-space one?  One is probably sufficient.